templates/install/openshift/enmasse-with-sasldb.yaml

```diff
@@ -40,6 +40,8 @@ objects:
               value: "${ADDRESS}"
             - name: CLUSTER_ID
               value: "${CLUSTER_ID}"
+            - name: PN_TRACE_FRM
+              value: '1'
```

Comment:
Perhaps we should remove this file, it is not used (or usable for that sake) anymore.

Reply:
done